core/services/cable_guy/api/manager.py

```diff
@@ -34,6 +34,11 @@ class EthernetInterface(BaseModel):
     info: Optional[InterfaceInfo]
 
 
+default_config = EthernetInterface(
+    name="eth0", configuration=InterfaceConfiguration(ip="192.168.2.2", mode=InterfaceMode.Unmanaged)
```

Comment:
Since this will break when running locally, could this at least be configured as an argument ?

Reply:
You mean when running directly on your machine, for example? 
It doesn't break because it uses "enpxsx" instead of "ethx". Ubuntu also moved back in 2016.

But even in this case, using it with an argument seems more reasonable. Adding.